grails-app/services/org/pih/warehouse/tableroapi/IndicatorDataService.groovy

```diff
@@ -18,7 +18,7 @@ class IndicatorDataService {
         Integer querySize = params.querySize ? params.querySize.toInteger() - 1 : 5
 
         // expirationSummary lists every expired item based on its month
-        List expirationSummary = []
+        List expirationSummary = new int[querySize]
```

Comment:
What's the point of doing it? This does not look like a grails way of initializing List.

Reply:
The point is to initialize the List with 0s, then empty positions will have 0 as value instead of null.

Reply:
I didn't realize that. TIL that you can do the same thing like this. 
```
List expirationSummary = [0] * querySize
```

Reply:
Yup I found similar solution, but wanted to test it before suggesting.